### PR TITLE
Mocha timeout disabled while opening browser

### PR DESCRIPTION
**Pull request by georgiy-abbasov**

### CODE REVIEW

File: test/functional/setup.js [hunk 1]
```diff
@@ -34,6 +37,9 @@ function openRemoteBrowsers () {
     return slConnector
         .connect()
         .then(function () {
+            return slConnector.waitForFreeMachines(3, WAIT_FOR_FREE_MACHINES_REQUEST_INTERVAL, 45);
```
Comment (AlexanderMoskovkin) [inline test/functional/setup.js]: move `3` and `45` to constants

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
@AlexanderMoskovkin, FPR

**Comment by AlexanderMoskovkin** [comment]
let's decrease test-case timeout to 30s

/r-

**Comment by georgiy-abbasov** [comment]
\cc @VasilyStrelyaev, check the comment, please
https://github.com/DevExpress/testcafe/pull/476/files#diff-15454f369c87c574b6885fdfb5bd6489R99

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by georgiy-abbasov** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by inikulin** [comment]
@AlexanderMoskovkin I have no idea what this PR fixes

**Comment by georgiy-abbasov** [comment]
@inikulin, mocha timeout affects the "before all" method, in other words it has the same timeout as tests, so functional tests task falls while connecting to Sauce Labs and starting VM.
Also, Sauce Labs virtual machines are configurated so the maximum running time is 30 minutes.
We had cases, when two machines have already worked for about 25 minutes and the third just started. The result - browsers are turned off in the middle of tests

**Comment by inikulin** [comment]
lgtm then

**Comment by inikulin** [comment]
@georgiy-abbasov  Can you list reviewers, please?

**Comment by AlexanderMoskovkin** [comment]
It was `ltgm` from me, @inikulin and @VasilyStrelyaev so I merge this

**Pull request merged**